### ISSUE TITLE
how do you visualize the frequency pictures

### ISSUE THREAD

**Issue by kiriiiia** [INVOKERer/DeepRFT#21]
In your paper,i see many analysis about frequency，so i want to know how to visualize the frequency picture，like high frequency，low frequency，or all together

**Comment by kiriiiia** [comment]
i’ll try，thankyou

**Comment by kiriiiia** [comment]
can you offer a more specific example or a piece code？thank you

**Comment by hzlbbfrog** [comment]
Hi, Dear INVOKERer. 
Thank you for your impressive research paper!
And I have the same question as kiriiiia.
Could you please release some code about the visualization of Fourier spectrum? 
e.g. in the figure 5, the channels of feature maps are different. I want to know how you handle this.
Maybe, emmm, I am not sure. Do a channel-wise average pooling for the frequency (= log(abs(fft(x)) + 1))?
Thank you so much!

**Comment by INVOKERer** [comment]
Yes, I use a channel-wise average pooling for the frequency (= log(abs(fft(x)) + 1)) and normalize it to [0, 1] .

**Comment by hzlbbfrog** [comment]
Dear INVOKERer,
Thank you for your quick reponse!
I will try it.
Best regards,
